core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/teradata/TeradataDatabaseConnectionService.java

```diff
@@ -27,4 +27,9 @@ public String getDriver(TeradataDatabaseConnection databaseConnection) {
     public Class<TeradataDatabaseConnection> appliesTo() {
         return TeradataDatabaseConnection.class;
     }
+
+    @Override
+    public String refactorLimitAndOffset(TeradataDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class